src/PAModelTests/UtilityTests.cs

```diff
@@ -36,29 +37,35 @@ public void TestNotEscaped()
         }
 
         [DataTestMethod]
-        // [DataRow("C:\\Foo\\Bar\\Baz", "C:\\Foo", "Bar\\Baz\\")]
-        // [DataRow("C:\\Foo\\Bar\\Baz", "C:\\Foo\\", "Bar\\Baz\\")]
-        // [DataRow("C:\\Foo\\Bar\\Baz\\", "C:\\Foo\\", "Bar\\Baz\\")]
-        // [DataRow("C:\\Foo\\Bar.msapp", "C:\\Foo", "Bar.msapp")]
-        // [DataRow("C:\\Foo\\Bar.msapp", "C:\\Foo\\", "Bar.msapp")]
-        // [DataRow("C:\\Foo\\Bar.msapp", "C:\\", "Foo\\Bar.msapp")]
-        [DataRow(@"C:\DataSources\JourneyPlanner|Sendforapproval.json", "C:\\", @"DataSources\JourneyPlanner|Sendforapproval.json")]
-        [DataRow(@"C:\DataSources\JourneyPlanner%7cSendforapproval.json", "C:\\", @"DataSources\JourneyPlanner%7cSendforapproval.json")]
-        // [DataRow(@"d:\app\Src\EditorState\Screen%252.editorstate.json",
-        //     @"d:\app", @"Src\EditorState\Screen%252.editorstate.json" )]
+        [DataRow(@"C:\Temp\DataSources\JourneyPlannerSendforapproval.json", @"C:\Temp", @"DataSources\JourneyPlannerSendforapproval.json")]
```

Comment:
I added the sub directory `Temp` to the path since this file will get written to the hard drive now.  I figured putting it in a folder called `Temp` would be better than having a directory called `DataSources` on the root of the hard drive. Will this cause any issues with an automated build/test?